README.md

```diff
@@ -113,13 +114,19 @@ You can deploy this project using:
 - Twitter: [@_naanahmed](https://twitter.com/_naanahmed_)
 - LinkedIn: [naanahmed](https://linkedin.com/in/naanahmed)
 
+👤 **Roodjino Cherilus**
+
+- GitHub: [roodjino](https://github.com/roodjinocherilus)
+- LinkedIn: [roodjino](https://www.linkedin.com/in/roodjinocherilus/)
+
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FUTURE FEATURES -->
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-- [ ] **Add ability to do Calculus problems**
+- **Add ability to do Calculus problems**
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you add some more meaningful features in the Future Feature section to make your readme more descriptive and professional 👍 😊

Reply:
Okay Thank you @HAMMAS-SALEEM. I will implement it.